Assets/Scripts/DayNightCycle/DayAndNightControl.cs

```diff
@@ -0,0 +1,246 @@
+﻿using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Animations;
+using UnityEditor;
+using Assets.Scripts.Services;
+using Assets.Scripts.Settings;
+
+namespace Assets.Scripts.DayNightCycle
+{
+    public class DayAndNightControl : MonoBehaviour
+    {
+        public GameObject moon;
+        public ReflectionProbe reflectionProbe;
+        public float sunTilt = -15f;
+        [SerializeField] private TimecycleProfile timecycle = null;
+        [SerializeField] private float sunRotationOffset = 0f;
+        [Tooltip("The amount of frames to wait before doing the next lighting update")]
+        [SerializeField] private int lightingUpdateInterval = 10;
+        public Material skyBoxPROCEDURAL;
+
+        [Range(0f, 24f)] public float currentTime;
+        public float CurrentTime01 => currentTime / 24;
+        public Slider TimeSlider  = null;
+        public Camera MoonCamera = null;
```

Comment:
Shouldn't the MoonCamera be disabled if you are within a time of day where the moon is not present? In your Update method, you are changing the FOV of the moon every frame, yet what is the point if the moon is not visible?

Reply:
I can put an if statement for the times when the moon is definitely below terrain and unobservable, then the FOV is only updated when the moon is visible

Reply:
> I can put an if statement for the times when the moon is definitely below terrain and unobservable, then the FOV is only updated when the moon is visible

Does this not also effect the sun?

Reply:
It does seem so from some quick testing, so i dont think we can disable this.